tests/eas/acceptance.py

```diff
@@ -84,6 +86,52 @@ def log_result(data, log_fh):
     logging.info(result_str)
     log_fh.write(result_str)
 
+class EasTest(LisaTest):
+    """
+    Base class for EAS tests
+    """
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        conf_file = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                 cls.conf_basename)
+
+        super(EasTest, cls)._init(conf_file, *args, **kwargs)
+
+    @memoized
+    def get_multi_assert(self, experiment, task_filter=""):
+        tasks = experiment.wload.tasks.keys()
+        return SchedMultiAssert(experiment.out_dir,
+                                self.te.topology,
+                                [t for t in tasks if task_filter in t])
+
+    @classmethod
+    def _experimentsInit(cls, *args, **kwargs):
+        super(EasTest, cls)._experimentsInit(*args, **kwargs)
+
+        if SET_IS_BIG_LITTLE:
+            try:
+                cls.target.write_value(
+                    "/proc/sys/kernel/sched_is_big_little", 1)
+            except TargetError:
+                # That flag doesn't exist on mainline-integration kernels, so
+                # don't worry if the file isn't present.
+                pass
+
+    def get_start_time(self, experiment):
```

Comment:
:+1: 
Although not required not, maybe we can add a dual `get_end_time(self. experiment)` just for consistency?